react/features/recording/components/LiveStream/StartLiveStreamDialog.web.js

```diff
@@ -237,18 +252,16 @@ class StartLiveStreamDialog
 
         switch (this.props._googleAPIState) {
         case GOOGLE_API_STATES.LOADED:
-            googleContent = (
-                <GoogleSignInButton
-                    onClick = { this._onGoogleSignIn }
-                    text = { t('liveStreaming.signIn') } />
+            googleContent = ( // eslint-disable-line no-extra-parens
```

Comment:
I think someone recently made changes so this linting error no longer happens. commit  1d99abc4

Reply:
Good catch, I didn't even try it, thanks!